pkg/logging/options.go

```diff
@@ -116,3 +148,82 @@ var LogDecision = map[string]Decision{
 	"LogFinishCall":         LogFinishCall,
 	"LogStartAndFinishCall": LogStartAndFinishCall,
 }
+
+// MapAllowedLevels allows to map a given level to a list of allowed level.
+// Convention taken from go-kit/level v0.10.0 https://godoc.org/github.com/go-kit/kit/log/level#AllowAll.
+var MapAllowedLevels = map[string][]string{
+	"DEBUG": {"INFO", "DEBUG", "WARN", "ERROR"},
+	"ERROR": {"ERROR"},
+	"INFO":  {"INFO", "WARN", "ERROR"},
+	"WARN":  {"WARN", "ERROR"},
+}
+
+// TODO: @yashrsharma44 - To be deprecated in the next release.
+func DecideHTTPFlag(flagDecision string, configYAML []byte) ([]Option, error) {
```

Comment:
This function currently uses both the flags `request.logging` and `log.request.decision` for enabling req. logging. Until we deprecate `log.request.decision`, we use this for using one of the flags for configuration, raise error if both are configured.

Reply:
This is correct behavior to me (:

Reply:
Cool